src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,199 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    public static final int FALSE_INDEX = -1;
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
     @Override
     public void add(T value) {
+        linkLast(value);
     }
 
     @Override
     public void add(T value, int index) {
+        areIndexInSizeRange(index);
+        linkReassignmentForAdd(value, index);
```

Comment:
there is no nee to create separate method linkReassignmentForAdd(), just move all code from it  here.